### PR TITLE
Get the global menu to work as soon as dynamo is open

### LINKED ISSUE #173: The BHoM is not loaded at startup, but when you place your first component on the canvas
#### Description:

I tried to ask here, what is the recommended way to solve this issue:
https://github.com/DynamoDS/Dynamo/issues/9830

### PR DESCRIPTION
**Pull request by adecler**
### Issues addressed by this PR

Closes #173 

Until now, we had to drop a component on the canvas for the Global menu to start working. This PR fixes that.

Note that global menu still doesn't work at all in 2020. Sadly this is a completely different problem and far from trivial (this is Autodesk after all) so will fix separately.

### Test files
Just open Dynamo and hit `Ctrl+Shift+B`. Easier test of your life 😉 

### Additional comments
I noticed that you don't need to drop a Revit component first anymore. Was it specific a a given version of Revit ? I couldn't find the issue talking about this.

### CODE REVIEW

**Comment by pawelbaran** [comment]
@adecler I have just ran some tests:
- in Dynamo Sandbox everything runs as explained in the PR
- in Dynamo for Revit:
    - the BHoM assemblies still load on the first component, not on startup (same as on `master`)
    - global menu does not work at all (same as on `master` - tbh I thought we do not have ctrl+shift+b at all in Dynamo until today 🙊)
    - in Revit 2020 + Dynamo 2.3, native component needs to be added as first, otherwise the UI blows up (same on `master`)

In short: all looks good in Dynamo Sandbox, but no progress on Dynamo Revit, which I guess is preferred by most of the users. I am not sure if that should be considered as the scope of this PR. Therefore I am leaving the comment instead review - happy to either approve it as Dynamo Sandbox PR or wait for Dynamo Revit being resolved in this PR too.

**Review by pawelbaran — APPROVED** [review]
As discussed offline: approving this PR as it improves Dynamo Sandbox and Dynamo Revit is meant to be handled in a separate thread.
